partcad/src/partcad/part_config_manufacturing.py

```
@@ -0,0 +1,49 @@
+
+
+class PartConfigManufacturing:
+    method: int | None
+
+    def __init__(self, final_config: dict) -> None:
+        manufacturing_config = final_config.get("manufacturing", {})
+        method_string = manufacturing_config.get("method", None)
+        self.method = _METHOD_MAP.get(method_string, METHOD_NONE)
+        if self.method == METHOD_NONE and method_string is not None:
+            pc_logging.error(
+                f"Unknown manufacturing method '{method_string}'. Supported methods: {list(_METHOD_MAP.keys())}."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)